doc/development/guide.rst

```diff
@@ -14,6 +14,12 @@ of PennyLane, plugins, and devices.
     :description: Installation and dependencies of the PennyLane source code using development mode.
     :link: guide/installation.html
 
+
+.. titlecard::
+    :name: Contribution guide
+    :description: How to get involved in the PennyLane community.
```

Comment:
Suggest making this three lines in vertical length if possible, so that the height of the card matches the other cards! Currently this card is only two lines long.